### PR TITLE
Updated help text for artifact checkboxes

### PR DESCRIPTION
**Pull request by nbhoski**
Updated help text for all test artifact checkboxes.
fix for #96 

PDF :
image

JUnit 
image
TAP : 
image
Stm : 
image

Cobertura code : 
image

Cobertura Model Coverage  : 
image

### CODE REVIEW

File: src/main/resources/com/mathworks/ci/RunMatlabTestsBuilder/help-pdfReportArtifact.html [hunk 1]
```diff
@@ -1,6 +1,7 @@
 <div>
 <br>
-Generate a test report in PDF format, <b>testreport.pdf</b>, and save it in the <b>matlabTestArtifacts</b> folder of the Jenkins workspace.
+Generate a test report in PDF format. By default, MATLAB names the artifact <b>testreport.pdf</b> and stores it in the <b>matlabTestArtifacts</b> folder of the project workspace. 
+To override the default artifact name and location, specify a path relative to the project folder in the <b>File path</b> box. If the text box is empty, MATLAB does not generate an artifact.
```
Comment (mw-hrastega) [inline src/main/resources/com/mathworks/ci/RunMatlabTestsBuilder/help-pdfReportArtifact.html:4]: In the note:
<b>Note:</b>This feature is not currently supported on MacOS platforms.

1) A white space is missing between Note: and This.
2) We can say macOS or Mac, but not MacOS.

File: src/main/resources/com/mathworks/ci/RunMatlabTestsBuilder/help-modelCoverageArtifact.html [hunk 2]
```diff
@@ -1,6 +1,9 @@
 <div>
-<br>Generate a Cobertura model coverage report, <b>coberturamodelcoverage.xml</b>, and save it in the <b>matlabTestArtifacts</b> folder of the Jenkins workspace.
-This report includes coverage results for Simulink&#174; models that are tested using Simulink Test&#8482;.<br>
-<br>Note: This feature requires a <b>Simulink Coverage&#8482;</b> license and is supported only in MATLAB R2018b or a newer release.<br>
+<br>
+Generate a model coverage report in Cobertura XML format. By default, MATLAB names the artifact <b>coberturamodelcoverage.xml</b> and stores it in the <b>matlabTestArtifacts</b> folder of the project workspace.
```
Comment (mw-hrastega) [inline src/main/resources/com/mathworks/ci/RunMatlabTestsBuilder/help-modelCoverageArtifact.html:3]: I would reorganize the text here (moving the third sentence in between the first and second).

Generate a model coverage report in Cobertura XML format. This artifact includes coverage results for Simulink&#174; models that are tested using Simulink Test&#8482;. By default, MATLAB names the artifact <b>coberturamodelcoverage.xml</b> and stores it in the <b>matlabTestArtifacts</b> folder of the project workspace.

File: src/main/resources/com/mathworks/ci/RunMatlabTestsBuilder/help-pdfReportArtifact.html (hunk 1 above)
Comment (nbhoski) [reply src/main/resources/com/mathworks/ci/RunMatlabTestsBuilder/help-pdfReportArtifact.html:4]: Done.

File: src/main/resources/com/mathworks/ci/RunMatlabTestsBuilder/help-modelCoverageArtifact.html (hunk 2 above)
Comment (nbhoski) [reply src/main/resources/com/mathworks/ci/RunMatlabTestsBuilder/help-modelCoverageArtifact.html]: Done.

**Comment by nbhoski** [comment]
I also realised that this may require changes too https://github.com/mathworks/jenkins-matlab-plugin/blob/master/CONFIGDOC.md#run-matlab-tests

**Pull request merged**